### PR TITLE
Feature/delete photos

### PR DESCRIPTION
**Pull request by mip92**

### CODE REVIEW

File: src/components/MyWorkplace/Pictures.tsx [hunk 1]
```diff
@@ -8,11 +8,14 @@ import $api from "../../http";
...
 import MyPicture from "./MyPicture";
-import {StateOpenInterface} from "./WorkplaceTable";
+import {StateOpenInterface} from "./OneMasterOrder";
+import {useDispatch} from "react-redux";
+import {deletePictures} from "../../actionCreators/workplaseActionCreators";
 
 interface PicturesProps {
     open: StateOpenInterface
     setOpen: Function
+    pictures: any
```
Comment (IvashchenkoCW) [inline src/components/MyWorkplace/Pictures.tsx]: any?

File: src/components/MyWorkplace/Pictures.tsx [hunk 2]
```diff
@@ -37,46 +40,28 @@ export interface pictureData {
...
     const [state, setState] = useState({});
     const handleChange = (event) => {
         setState({...state, [event.target.name]: event.target.checked});
     };
 
-
     const onDelete = async () => {
         try {
-            let arr:string[] = []
+            let arr: string[] = []
```
Comment (IvashchenkoCW) [inline src/components/MyWorkplace/Pictures.tsx]: arr???

File: src/components/MyWorkplace/Pictures.tsx [hunk 3]
```diff
@@ -37,46 +40,28 @@ export interface pictureData {
...
             const entries = Object.entries(state);
-            entries.forEach((p)=>{
+            entries.forEach((p) => {
```
Comment (IvashchenkoCW) [inline src/components/MyWorkplace/Pictures.tsx:38]: I don't understand the logic

File: src/components/MyWorkplace/Pictures.tsx [hunk 4]
```diff
@@ -89,28 +74,32 @@ const Pictures: React.FC<PicturesProps> = ({open, setOpen}) => {
                 <DialogContent>
                     <DialogContentText id="alert-dialog-description">
                         <FormControl component="fieldset">
-                            {urls && urls.map((u, key) =>
+                            {pictures && pictures.map((picture, key) =>
                                 <div>
                                     <FormControlLabel
                                         key={key}
```
Comment (IvashchenkoCW) [inline src/components/MyWorkplace/Pictures.tsx]: do not use index as key

Comment (IvashchenkoCW) [reply src/components/MyWorkplace/Pictures.tsx]: use Id

File: src/store/reducers/adminMasterReducer.ts [hunk 5]
```diff
@@ -20,7 +20,7 @@ export const adminMasterReducer = (state = initState, action: AdminMasterAction)
             return {...state, masters: state.masters.filter(master => master.id !== action.payload.payload.id)}
         case AdminMastersActionTypes.APPROVE_MASTER:
             return {...state, masters: state.masters.map(m=>{
-                if (m.id==action.payload.payload.id)return {...m, isApproved:action.payload.payload.isApproved}
+                if (m.id==action.payload.payload.id) return {...m, isApproved:action.payload.payload.isApproved}
```
Comment (IvashchenkoCW) [inline src/store/reducers/adminMasterReducer.ts]: m?

Comment (IvashchenkoCW) [reply src/store/reducers/adminMasterReducer.ts]: ==

File: src/store/reducers/workplaceReducer.ts [hunk 6]
```diff
@@ -51,6 +67,7 @@ export interface Order {
     master: Master,
     user: User,
     master_busyDate: MasterBusyDate,
+    orderPictures: OrderPicture[] | []
```
Comment (IvashchenkoCW) [inline src/store/reducers/workplaceReducer.ts]: OrderPicture[] | [] ?

File: src/types/mainInterfacesAndTypes.tsx [hunk 7]
```diff
@@ -37,6 +37,12 @@ export type BooleanPayload = {
 export type OrdersPayload = {
     payload: Order[]
 }
+export type DelPicturesPayload = {
```
Comment (IvashchenkoCW) [inline src/types/mainInterfacesAndTypes.tsx]: del?

File: src/store/reducers/workplaceReducer.ts (hunk 6 above)
Comment (mip92) [reply src/store/reducers/workplaceReducer.ts]: if there is no image in the order, the backend returns an empty array

**Pull request merged**